### DISCUSSION TITLE
Change repository url

### DISCUSSION THREAD

**Discussion by sunny-azu-red** [tteck/Proxmox#2716] [Q&A]
I want to install homepage LXC but not from their official page repo.
Instead i want it downloaded from my forked repository, because I made some changes that they do not wish to implement.

How would I go about using the helper script and changing the url of the repository?
Is it even possible?

Thanks!

**Comment by buczi11** [comment]
> I want to install homepage LXC but not from their official page repo. Instead i want it downloaded from my forked repository, because I made some changes that they do not wish to implement.
> 
> How would I go about using the helper script and changing the url of the repository? Is it even possible?
> 
> Thanks!

Hi, 
start with the script related to homepage and review its content, I think it will help you
https://github.com/tteck/Proxmox/blob/main/install/homepage-install.sh

**Reply by sunny-azu-red** [reply]
but you need to call https://raw.githubusercontent.com/tteck/Proxmox/main/ct/homepage.sh in order to install the LXC, the link you provided is not helpful

**Reply by buczi11** [reply]
hmm,
you would have to do a fork of the repository

**Reply by sunny-azu-red** [reply]
> of the repository

which one? 😁
